src/main/java/edu/kis/powp/jobs2d/command/manager/parsers/Parser.java

```diff
@@ -0,0 +1,5 @@
+package edu.kis.powp.jobs2d.command.manager.parsers;
+
+public interface Parser {
+    void parseToDriverCommand();
```

Comment:
This interface does not make any sense. The client can only parse and cannot get anything in return!?